src/backend/SecOperation.c

```diff
@@ -169,6 +169,12 @@ bool BSL_SecOper_IsBIB(const BSL_SecOper_t *self)
     return self->_service_type == BSL_SECBLOCKTYPE_BIB;
 }
 
+BSL_PolicyAction_e BSL_SecOper_GetFailureCode(const BSL_SecOper_t *self)
```

Comment:
The name "failure code" vs the type name "policy action" seem to be in disagreement. This goes along with documentation, what is this and how should it be used (if at all) by either PPs or SCs.